cmd/gen-go-sample/main.go

```diff
@@ -224,6 +225,77 @@ func (g *generator) genSample(ifaceName string, methConf GAPICMethod, regTag str
 	}
 	g.imports[inSpec] = true
 
+	initInfo, err := g.getInitInfo(inType, methConf, valSet)
+	argNames := initInfo.argNames
+	argTrees := initInfo.argTrees
+	flagNames := initInfo.flagNames
+	files := initInfo.files
+	itree := initInfo.reqTree
+
+	if err := g.sampFuncAndClient(serv.GetName(), meth.GetName(), initInfo, regTag); err != nil {
+		return err
+	}
+
+	// Set up the request
+	{
+		var buf bytes.Buffer
```

Comment:
Why are we using a new buffer instead of just using the `g.pt.Writer()`? The buffer gets written to the `g.pt` later anyways.

Reply:
It's the prependLines on L261 I think. It modifies the buffer afterwards.

Reply:
Yes! We need to insert those "//" to some of the lines here.